src/JsonSchema/Rfc3339.php

```diff
@@ -23,7 +23,9 @@ public static function createFromString($string)
         $microseconds = $matches[2] ?: '.000000';
         $timeZone = 'Z' !== $matches[3] ? $matches[4] . ':' . $matches[5] : '+00:00';
 
-        $dateTime = \DateTime::createFromFormat('Y-m-d\TH:i:s.uP', $dateAndTime . $microseconds . $timeZone, new \DateTimeZone('UTC'));
+        $dateFormat = strpos($dateAndTime, 'T') === false ? 'Y-m-d H:i:s.uP' : 'Y-m-d\TH:i:s.uP';
```

Comment:
(Although that said, I don't think that this is a huge issue, because timezone codes aren't currently allowed, so things like 'PST' will not be an issue at present. So if you'd rather leave it that way it is, that's fine too.)